src/Altinn.App.Core/Internal/Pdf/PdfService.cs

```diff
@@ -103,15 +108,15 @@ public async Task<Stream> GeneratePdf(Instance instance, string taskId, Cancella
 
         TextResource? textResource = await GetTextResource(instance, language);
 
-        return await GeneratePdfContent(instance, taskId, language, textResource, ct);
+        return await GeneratePdfContent(instance, language, ct, textResource, isPreview);
     }
 
     private async Task<Stream> GeneratePdfContent(
         Instance instance,
-        string taskId,
         string language,
-        TextResource? textResource,
-        CancellationToken ct
+        CancellationToken ct,
+        TextResource? textResource = null,
+        Boolean? isPreview = false
```

Comment:
CancellationToken last. No need for default values for compatibility with `private` methods